assets/app/styles/_core.less

```diff
@@ -301,6 +301,14 @@ html, body {
   }
 }
 
+label.required:before {
+  content: '* ';
+  // Use position: absolute instead of negative margin so the asterisk shows
+  // up when overflow is hidden.
+  position: absolute;
+  left: 10px;
```

Comment:
doesn't this inset 10px from the left? does this presume there's at least 10px+width('*') of left padding?